docs/modules/ROOT/pages/documentation/ecosystem/vscode-extension/payara-server.adoc

```diff
@@ -52,6 +52,42 @@ Right-click on the application and select *Run on Payara Server* to deploy the a
 
 image::vscode-extension/payara-server/deploy-application.png[alt=Deploy application, width="60%"]
 
+Alternatively application can be deployed from *Java Projects* interface:
+
+image::vscode-extension/payara-server/java-projects-deploy.png[alt=Deploy application, width="60%"]
+
 The deploy application can be opened in the browser, undeployed or disabled by right-clicking on the application:
 
-image::vscode-extension/payara-server/list-application.png[alt=Open application homepage, width="60%"]
\ No newline at end of file
+image::vscode-extension/payara-server/list-application.png[alt=Open application homepage, width="60%"]
+
+[[deployment-settings]]
+== Deployment Settings
+
+The Auto Deploy and Hot Deploy mode are helpful for developers to run and test 
+an application immediately after making changes to its sources without 
+restarting the Server or manual redeployment to maximize your productivity.
+
+[[hot-deploy]]
+=== Hot Deploy
+
+_Since *5.201*&nbsp;_ Payara Server Community provides the Hot Deployment capability by boosting the performance 
```

Comment:
```suggestion
_Since *5.201*&nbsp;_

Payara Server provides the Hot Deployment capability, which boosts the performance
```

The service team suggests that we don't mention the edition name so that it can be easily copied to both editions.